<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>779</width>
    <height>391</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Attack Configuration</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>Among Us</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Data Collection</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QCheckBox" name="store_resp">
         <property name="font">
          <font>
           <family>Among Us</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Store Raw Response</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="store_timing">
         <property name="font">
          <font>
           <family>Among Us</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Store Timing Info</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="store_length">
         <property name="font">
          <font>
           <family>Among Us</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Store Content Length</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="show_filter">
       <property name="font">
        <font>
         <family>Among Us</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Store Response If:</string>
       </property>
       <property name="textFormat">
        <enum>Qt::PlainText</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="store_resp_if">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Among Us</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>True</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <family>Among Us</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Terminate Attack If:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="terminate_if">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Among Us</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>stored_response.status_code &gt; 499</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="ok_btn">
       <property name="font">
        <font>
         <family>Among Us</family>
         <pointsize>26</pointsize>
        </font>
       </property>
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
